build/teamcity/cockroach/ci/tests/unit_tests_impl.sh

```diff
@@ -3,6 +3,13 @@
 set -xeuo pipefail
 
 bazel build //pkg/cmd/bazci --config=ci
-$(bazel info bazel-bin --config=ci)/pkg/cmd/bazci/bazci_/bazci --config=cinolint --config=simplestamp --compilation_mode=fastbuild \
+
+if tc_release_branch; then
+  # enable up to 2 attempts per test executable to report flakes but only on release branches (i.e., not staging)
```

Comment:
3 attempts?

Reply:
Right. I think I meant to say "retries"; fixing.